apps/src/blockly/customBlocks/googleBlockly/spritelabBlocks.js

```diff
@@ -1,5 +1,102 @@
+import _ from 'lodash';
+import {SVG_NS} from '@cdo/apps/constants';
+import Button from '@cdo/apps/templates/Button';
+
 // This file contains customizations to Google Blockly Sprite Lab blocks.
 
+const dummyInputType = 5;
```

Comment:
Can you add a comment explaining what this field means?

Reply:
Glad you asked, as I meant to clean this up!

These types are defined here: https://github.com/google/blockly/blob/master/core/inputs/input_types.ts

We can actually add this to the wrapper and access it directly instead, which feels a whole lot less fragile:

https://github.com/code-dot-org/code-dot-org/pull/52842/commits/741f10c3eaa6f79542d4ddd231c69a9a36ead506